doc/deployment-integration/local-deployment/LOCAL.MD

```diff
@@ -824,6 +824,188 @@ with the TMForum-APIs directly.
+    # configure the path inside the keyloak container, where the signing key-store is located
+    storePath: /trust-store/keystore.p12
+    # password of the keystore
+    storePassword: password
+    # alias of the key inside the store
+    keyAlias: test-keystore
+    # password of the signing key
+    keyPassword: password
+    # algorithm of the key
+    keyAlgorithm: RS256
+    # did that the key belongs to
+    did: did:elsi:VATDE-1234567
+    # the store and its contents
+    keystore:
+      keystore.p12: <THE_KEYSTORE_BASE_&$>
+```
+This will load the keystore into a secret and configures the [realm](../../../charts/data-space-connector/templates/realm.yaml) to use did:elsi and load the key accordingly. 
+
+2. [Mount the store to keycloak](../../../k3s/consumer-elsi.yaml#L88-L105)
+```yaml
+  # mount the secret-volume
+  extraVolumeMounts:
+    - name: elsi-trust-store
+      # needs to match the storePath above
+      mountPath: /trust-store
+  ....
+  # load the secret into a volume
+  extraVolumes:
+    - name: elsi-trust-store
+      secret:
+        secretName: elsi-secret
+        defaultMode: 0755
+```
+3. [Add the jades-issuer via init-container](../../../k3s/consumer-elsi.yaml#L45)
+```yaml
+    # add the init container that copies the plugin to the shared folder
+    - name: install-jades-issuer
+      image: quay.io/fiware/keycloak-jades-vc-issuer:1.2.0
+      imagePullPolicy: Always
+      volumeMounts:
+        - name: providers
+          mountPath: /target
+  # mount provider folder into keycloak 
+  extraVolumeMounts:
+    - name: providers
+      mountPath: /opt/bitnami/keycloak/providers/
+  # create an empty dir 
+  extraVolumes:
+    - name: providers
+      emptyDir: {}
+```
+
+### The Provider
+
+Since the validation of [JAdES Signatures](https://www.etsi.org/deliver/etsi_ts/119100_119199/11918201/01.01.01_60/ts_11918201v010101p.pdf) is quite complex,
+the verifier does not fully validate itself. Instead, it requests validation at the [dss-validation-service](https://github.com/wistefan/dss-validation-service).
+The service uses the [Digital Signature Service Library](https://github.com/esig/dss/tree/master), that is provided as a [Digital Building Block by the European Comission](https://ec.europa.eu/digital-building-blocks/sites/display/DIGITAL/Digital+Signature+Service+-++DSS).
+To do so, the deployment needs to be changed as following:
+1. [Enable dss](../../../k3s/provider-elsi.yaml#L84)
+```yaml
+dss:
+  # enable the dss-validation-service
+  enabled: true
+  # port of the health check endpoint.
+  health: 
+    port: 8080
+  # in order to validate credentials, trust sources need to be configured
+  trust:
+    # trust sources can be [eIDAS Trusted Lists](https://eidas.ec.europa.eu/efda/trust-services/browse/eidas/tls)
+    lists:
+      # refresh interval of the list 
+      - refreshSeconds: 2
+        # url of the concrete list to be used
+        url: "https://ec.europa.eu/tools/lotl/eu-lotl.xml"
```

Comment:
The certificates used in the local setup are signed using some test certificate on this list or am I misunderstanding something?

Reply:
Ah no sorry just saw that for the local deployment a store is handed in the dss service